gustaf/vertices.py

```diff
@@ -55,7 +56,7 @@ def __init__(
 
     def process(
```

Comment:
Thanks for fixing typos ;) This experimental function will retire soon, but it can now retire in peace, knowing it can process stuff